src/scrabBag.js

```diff
@@ -0,0 +1,21 @@
+export default function scrabBag(theInput, tileDistro = {}) {
+  const inputArray = []
+
+  if (tileDistro === null) {
```

Comment:
`tileDistro` should never be null since it is initialized in the parameter list, above - this check is unnecessary.